### ISSUE TITLE
File Extension for In Progress Downloads

### ISSUE THREAD

**Issue by hl2guide** [erickutcher/httpdownloader#124]
Please consider having a different file extension for files that are in progress.

Some examples of other programs that do this:
MEGASync: .mega
Google Chrome: .part

Maybe something like:
- .download
- .httpd

That way without looking at the application we can see in Windows Explorer which files are in progress and which have completed.

**Comment by erickutcher** [comment]
I considered this before, but I don't like having extra file extensions. The compromise was to use a temporary download folder which can be enabled in the Advanced options.

**Comment by maliayas** [comment]
I also agree with this feature request. It also matches with native browser downloads. It is also semantically more correct; e.g. a half downloaded pdf file is not a pdf file. Something like a .part extension helps here a lot. That way, only by staying in the file explorer, I can tell if a file is fully downloaded or not.

I'd love it if you reconsider this feature request.

**Comment by erickutcher** [comment]
Maybe in the future. It's doable, but it's a lot of work and I kind of want to be done with this program. I have too many other things that I need to work on and can't devote as much time to it. But I'll consider it as an option.

**Comment by maliayas** [comment]
Sure, I see. Thanks a lot. Shall we at least keep the issue open until it's implemented by a contributor?
